README.md

```diff
@@ -10,122 +10,108 @@ Android Auto for IDrive
 
 User Guide
 ----------
 
-![Phone Connection](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-connection.png)
+| | |
+|--|--|
+| **Connecting to the car** ||
+After connecting the phone to the car, the official Connected app should show this car icon in the status bar. When this icon appears, this app should connect and add its functionality to the car. See [this guide](https://hufman.github.io/AndroidAutoIdrive/connection.html) for tips on improving the connection reliability of the Connected app.|![Phone Connection](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-connection.png)|
+| **Seeing the connected apps** ||
+|After all the apps are connected, a bunch of new entries will show up in the car's Connected menu. Besides the official Calendar and Connected apps, there should be a new Audioplayer icon and a book icon with no label. This book icon is the Notifications app, if enabled.|![Phone App List](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-phoneapps.jpg)|
+| **Seeing the music/media apps** ||
+|Several new entries will be added to the Media section of the control screen. The Audioplayer icon is the one with the main functionality, while the other displayed apps above the Audioplayer are quick shortcuts to switch playback to the respective apps. This screenshot also shows the official Spotify app at the bottom of the list.|![Music App List](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-medialist.jpg)|
+
+
+Implemented Features
+--------------------
+
+<details>
+<summary>🎙 Voice Assistants</summary>
 
-After connecting the phone to the car, the official Connected app should show this car icon in the status bar. When this icon appears, this app should connect and add its functionality to the car.
+- Any voice assistant installed on the phone is added as a Connected App entry
+- Google Assistant works the best, but Alexa, Bixby, and Cortana are also compatible
+- The voice assistant app can be set as a hardware shortcut button for convenience
 
-See [this guide](https://hufman.github.io/AndroidAutoIdrive/connection.html) for tips on improving the connection reliability of the Connected app.
+</details>
 
-![Phone App List](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-phoneapps.jpg)
+<details>
+<summary>🔔 Phone Notifications</summary>
 
-After all the apps are connected, a bunch of new entries will show up in the car's Connected menu. Besides the official Calendar and Connected apps, there should be a new Audioplayer icon and a book icon with no label. This book icon is the Notifications app, if enabled.
+- Popup about new notifications (on IDrive 4 or older)
+- Supports Dismiss, Mark As Read, replies or other notification actions
 
-![Music App List](https://hufman.github.io/AndroidAutoIdrive/images/screenshot-medialist.jpg)
+</details>
+<details>
+<summary>🎵 Control of Android Auto compatible music apps</summary>
```

Comment:
Perhaps just "Control of your music apps", since it can control more than just the apps that declare Android Auto support

Reply:
Yep, on a side note I think the AndroidAuto link is kind of misleading in other parts as well (e.g. people will think you can do most of the AndroidAuto stuff like control your phone, waze, etc.) but it's not quite what the app does 🤔